app/models/account.rb

```diff
@@ -26,6 +26,7 @@ def price_groups
   accepts_nested_attributes_for :account_users
 
   scope :active, lambda {{ :conditions => ['expires_at > ? AND suspended_at IS NULL', Time.zone.now] }}
+  scope :filter_global_account_types, -> { where("accounts.type in (:global)", { global: global_account_types.map(&:to_s) }) }
```

Comment:
I feel like `filter_global_account_types` almost implies that it filters them out. What would you think of simply `:global`? Also, is the `to_s` necessary? I would expect AR to take care of that for you.